src/BaseCharts.js

```diff
@@ -88,18 +137,78 @@ export function generateChart(chartId, chartType) {
   }
 }
 
-export const Bar = generateChart('bar-chart', 'bar')
+export const Bar = generateChart(
+  'bar-chart',
+  'bar',
+  [BarElement],
+  BarController,
+  [LinearScale, CategoryScale]
```

Comment:
Type of chart doesn't guarantee using of these scales and elements. So we should pass only controller type.

Reply:
fixed